restcomm/restcomm.rvd/src/main/webapp/languages/es.json

```diff
@@ -262,14 +263,14 @@
 	"formDefaultValueLabel": "Valor por defecto",
 	"formDefaultValuePlaceholder": "Bienvenido al IVR clinic. Un gusto verle.",
 	"formDefaultValueTooltip": "Escriba un valor por defecto opcional para el campo. Será pre-llenado con este valor.",
-	"formAppParamsRequiredLabel": "Necesario",
+	"formAppParamsRequiredLabel": "Requerido",
 	"formAppParamsRequiredTooltip": "Es necesario este campo?  Su aplicación tendría que manejar correctamente la ausencia de este campo.",
 	"formDescriptionLabel": "Descripción *",
 	"formDescriptionPlaceholder": "Este es el primer mensaje que la persona que recibe la llamada escuchará. Deles la bienvenida e infórmeles que viene a continuación.",

```

Comment:
Grammar error found. Change for:
"formDescriptionPlaceholder": "Este es el primer mensaje que la persona que recibe la llamada escuchará. Dele la bienvenida e infórmele que viene a continuación.",